app/models/event.rb

```diff
@@ -221,7 +221,7 @@ def set_created_email
 
     def eventdate_valid?
       if representative_date < Account.magic_date
-       errors.add(:representative_date, "Requested date out of range")
```

Comment:
Is eventdate_valid? used anywhere? I did a repo search to find out what this is used for and the only instance I found was this one lol